File: 백준/Bronze/30999. 민주주의/민주주의.cs
using System;

class Program
{
    static void Main()
    {
        string[] input = Console.ReadLine().Split(' ');
        int N = int.Parse(input[0]);
        int M = int.Parse(input[1]);
        int result = 0;

        for (int i = 0; i < N; i++)
        {
            string opinion = Console.ReadLine();
            int agreeCount = 0;

            foreach (char c in opinion)
            {
                if (c == 'O') agreeCount++;
            }

            if (agreeCount >= (M / 2) + 1)
            {
                result++;
            }
        }

        Console.WriteLine(result);
    }
}

File: 백준/Bronze/30999. 민주주의/민주주의.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int N = scanner.nextInt();
        int M = scanner.nextInt();
        scanner.nextLine(); // 버퍼 비우기
        int result = 0;

        for (int i = 0; i < N; i++) {
            String opinion = scanner.nextLine();
            int agreeCount = 0;

            for (char c : opinion.toCharArray()) {
                if (c == 'O') agreeCount++;
            }

            if (agreeCount >= (M / 2) + 1) {
                result++;
            }
        }

        System.out.println(result);
        scanner.close();
    }
}
